### PR TITLE
fix ideas order in v2

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

File: assembl/tests/views_tests/test_graphql_ideas.py [hunk 1]
```diff
@@ -41,10 +41,10 @@ def test_graphql_get_all_ideas(graphql_request,
...
     assert res.data['rootIdea']['id'] is not None
     assert len(res.data['ideas']) == 4
-    root_idea = res.data['ideas'][0]
-    first_idea = res.data['ideas'][1]
-    second_idea = res.data['ideas'][2]
-    third_idea = res.data['ideas'][3]
+    first_idea = res.data['ideas'][0]
+    second_idea = res.data['ideas'][1]
+    third_idea = res.data['ideas'][2]
+    root_idea = res.data['ideas'][3]
```
Comment (cedricmessiant) [inline assembl/tests/views_tests/test_graphql_ideas.py]: so root idea is always the last in order ?

Comment (vincentfretin) [reply assembl/tests/views_tests/test_graphql_ideas.py]: yes, because order is None

File: assembl/graphql/schema.py [hunk 2]
```diff
@@ -997,7 +997,9 @@ def resolve_ideas(self, args, context, info):
         root_idea_id = discussion.root_idea.id
         descendants_query = model.get_descendants_query(
             root_idea_id, inclusive=True)
-        query = query.filter(model.id.in_(descendants_query)
+        query = query.outerjoin(
```
Comment (ayyazdaniaryan) [inline assembl/graphql/schema.py]: I suspect that this is not fully right. There are many idea links that have both a null source and a null target, or a source and null target. But I think it's correct that the order is an ascending number. @maparent Maybe could chime in?

Comment (vincentfretin) [reply assembl/graphql/schema.py]: @ayyazdaniaryan I do exactly the same thing that in the rest api. It's a left outer join, and the order by is on both IdeaLink.order and Idea.creation_date

**Comment by vincentfretin** [comment]
Can I merge @ayyazdaniaryan ?

File: assembl/graphql/schema.py (hunk 2 above)
Comment (ayyazdaniaryan) [reply assembl/graphql/schema.py]: Alright. All good with me then.

Comment (vincentfretin) [reply assembl/graphql/schema.py]: In the rest api : https://github.com/assembl/assembl/blob/68f3348cfe35de7b962352303b61e893b26584a6/assembl/views/api/idea.py#L125-L130

**Comment by ayyazdaniaryan** [comment]
Si all good with me.

**Pull request closed without merging**